src/store/modules/HardwareStatus/FanStore.js

```diff
@@ -29,6 +29,7 @@ const FanStore = {
           partNumber: PartNumber,
           serialNumber: SerialNumber,
           identifyLed: LocationIndicatorActive,
+          ledStatus: LocationIndicatorActive,
```

Comment:
We can use identifyLed without creating ledStatus.

Reply:
each LED is attached with unique ledStatus, so mentioned separately.